src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdaterTest.java

```diff
@@ -0,0 +1,409 @@
+        given(eventResponse.getEventId()).willReturn("eventId");
+        given(eventResponse.getToken()).willReturn("token");
+        given(coreCaseDataApi.startForCaseworker(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString()))
+            .willReturn(eventResponse);
+        given(coreCaseDataApi.submitForCaseworker(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyBoolean(),
+            any(CaseDataContent.class)
+        )).willReturn(CaseDetails.builder().id(1L).build());
+        given(exceptionRecordFinalizer.finalizeExceptionRecord(anyMap(), anyLong())).willReturn(originalFields);
+
+        // when
+        ProcessResult res = ccdCaseUpdater.updateCase(
+            exceptionRecord,
+            configItem,
+            true,
+            "idamToken",
+            "userId",
+            existingCase
+        );
+
+        // then
+        assertThat(res.getErrors()).isEmpty();
+        assertThat(res.getWarnings()).isEmpty();
+        assertThat(res.getExceptionRecordData()).isEqualTo(originalFields);
+    }
+
+    @Test
+    void should_update_case_ignore_warnings() throws Exception {
+        // given
+        successfulUpdateResponse = new SuccessfulUpdateResponse(caseUpdateDetails, asList("warning1"));
+        given(caseUpdateClient.updateCase("url", existingCase, exceptionRecord, "token"))
+            .willReturn(successfulUpdateResponse);
+        given(existingCase.getCaseTypeId()).willReturn("caseTypeId");
+        given(eventResponse.getEventId()).willReturn("eventId");
+        given(eventResponse.getToken()).willReturn("token");
+        given(coreCaseDataApi.startForCaseworker(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString()))
+            .willReturn(eventResponse);
+        given(coreCaseDataApi.submitForCaseworker(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyBoolean(),
+            any(CaseDataContent.class)
+        )).willReturn(CaseDetails.builder().id(1L).build());
+        given(exceptionRecordFinalizer.finalizeExceptionRecord(anyMap(), anyLong())).willReturn(originalFields);
+
+        // when
+        ProcessResult res = ccdCaseUpdater.updateCase(
+            exceptionRecord,
+            configItem,
+            true,
+            "idamToken",
+            "userId",
+            existingCase
+        );
+
+        // then
+        assertThat(res.getErrors()).isEmpty();
+        assertThat(res.getWarnings()).isEmpty();
+        assertThat(res.getExceptionRecordData()).isEqualTo(originalFields);
+    }
+
+    @Test
+    void should_update_case_not_ignore_warnings() throws Exception {
```

Comment:
I think in general it would be nice to follow the pattern which we follow in other test classes

Reply:
I tried to guess which pattern you mean